From 0fe8ebf1d60f69883e474f8665a4db8e08f7e06d Mon Sep 17 00:00:00 2001
From: Suresh Gupta <suresh.gupta@xilinx.com>
Date: Thu, 7 Feb 2019 15:36:51 +0530
Subject: [PATCH 0503/1851] media: xilinx: multi-scaler: Replace dev_info with
 dev_dbg if all jobs are not ready

commit cd19b848bfb5d3d4d4931a1df9f95800108f2378 from
https://github.com/Xilinx/linux-xlnx.git

This patch changes the dev_info() call to dev_dbg() in
xm2msc_alljob_ready() to print out the information that which opened
channel is not ready yet, because this kind of information is only
useful for debugging.

Signed-off-by: Suresh Gupta <suresh.gupta@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-multi-scaler.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-multi-scaler.c b/drivers/media/platform/xilinx/xilinx-multi-scaler.c
index 89b0ed37d4d1..225b6a892eee 100644
--- a/drivers/media/platform/xilinx/xilinx-multi-scaler.c
+++ b/drivers/media/platform/xilinx/xilinx-multi-scaler.c
@@ -970,8 +970,8 @@ static bool xm2msc_alljob_ready(struct xm2m_msc_dev *xm2msc)
 		chan_ctx = &xm2msc->xm2msc_chan[chan];
 
 		if (!xm2msc_job_ready((void *)chan_ctx)) {
-			dev_info(xm2msc->dev, "chan %d not ready\n",
-				 chan_ctx->num);
+			dev_dbg(xm2msc->dev, "chan %d not ready\n",
+				chan_ctx->num);
 			return false;
 		}
 	}
-- 
2.31.1

